cypress/e2e/reviewRequests.spec.ts

```diff
@@ -108,127 +157,510 @@ describe("Review Requests", () => {
+        "not.be.disabled"
+      )
+      cy.contains(`${TEST_PAGE_MOVE}.md`)
+        .parent()
+        .contains(TARGET_COLLECTION)
+        .should("exist")
+      cy.contains(`${TEST_PAGE_DELETE}.md`).should("exist")
+      getReviewRequestOverviewPageEditButton(`${TEST_PAGE_DELETE}.md`).should(
+        "be.disabled"
+      )
+      cy.contains(`new ${TEST_PAGE_RENAME}.md`).should("exist")
+      getReviewRequestOverviewPageEditButton(
+        `new ${TEST_PAGE_RENAME}.md`
+      ).should("not.be.disabled")
     })
   })
-  describe.skip("has approved review requests", () => {
+  describe("has approved review requests", () => {
     before(() => {
-      addAdmin(E2E_EMAIL_COLLAB.email)
-      // NOTE: Create a review request if none exists
-      listReviewRequests().then((requests) => {
-        if (!requests || !requests.length) {
-          createReviewRequest(
-            MOCK_REVIEW_TITLE,
-            [E2E_EMAIL_COLLAB.email],
-            MOCK_REVIEW_DESCRIPTION
-          ).then((id) => approveReviewRequest(id))
-        }
+      addAdmin(E2E_EMAIL_ADMIN_2.email)
+      closeReviewRequests()
+      cy.setEmailSessionDefaults("Email admin")
+      cy.actAsEmailUser(E2E_EMAIL_ADMIN_2.email, "Email admin")
+      editUnlinkedPage(
+        "faq.md",
+        "some approved review requests content",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      createReviewRequest(
+        MOCK_REVIEW_TITLE,
+        [E2E_EMAIL_ADMIN.email],
+        MOCK_REVIEW_DESCRIPTION
+      ).then((id) => {
+        cy.actAsEmailUser(E2E_EMAIL_ADMIN.email, "Email admin")
+        approveReviewRequest(id)
       })
     })
-    it("should have the review request approved alert stating that editing is disabled when on the workspace", () => {
-      // Arrange
-      // Act
-      // Assert
+    beforeEach(() => {
+      // NOTE: We want to start this test suite on a clean slate
+      // and prevent other tests from interfering
+      cy.setupDefaultInterceptors()
+      cy.setEmailSessionDefaults("Email admin")
+      visitE2eEmailTestRepo()
     })
-    it("should show the review request as being approved on the dashboard", () => {
+    it("should have the review request approved alert stating that editing is disabled when on the workspace", () => {
       // Arrange
+      const RR_INTERCEPTOR = "getReviewRequest"
+      cy.intercept("GET", "**/summary").as(RR_INTERCEPTOR)
+
       // Act
+      cy.visit(`/sites/${E2E_EMAIL_TEST_SITE.repo}/workspace`)
+      cy.wait(`@${RR_INTERCEPTOR}`)
+
       // Assert
+      // Toast
+      cy.contains("There is currently an approved review request!")
+      // Redirect to dashboard
+      cy.url().should("include", `/sites/${E2E_EMAIL_TEST_SITE.repo}/dashboard`)
+      // Tag
+      cy.contains("Approved")
     })
     it("should prevent edits while the request is not merged", () => {
       // Arrange
+
       // Act
+      cy.visit(`/sites/${E2E_EMAIL_TEST_SITE.repo}/editPage/example-page.md`)
```

Comment:
could we additionally check that pointer events are disabled? (lmk if this is hard)

Reply:
Added in e47b68e4